draft-ietf-rats-msg-wrap.md

```diff
@@ -246,6 +254,28 @@ func CMWTypeSniff(b []byte) (CMW, error) {
 }
 ~~~
 
+## CMW Collections {#cmw-coll}
+
+Layered attesters and composite devices ({{Sections 3.2 and 3.3 of -rats-arch}}) generate evidence that consists of multiple parts.
+
+For example, in data center servers, it is not uncommon for separate Attesting Environments (AE) to serve a subsection of the entire machine.
+One AE might produce Evidence including Claims that represent what was booted on the main CPU, or on a SmartNIC plugged into a PCIe slot, or on the machine's GPU.
+
+To address the composite Attester use case, this document defines a CMW "collection" as a container that holds several CMW items, each with a label that is unique within the scope of the collection.
+
+The CMW collection ({{fig-cddl-collection}}) is defined as a CBOR map or JSON object with CMW values.
+The position of a `cmw` entry in the `cmw-collection` is not significant.
+Instead, the labels identify a conceptual message that, in the case of a composite attester, should typically correspond to a component of a system.
```

Comment:
"component of a system" seems too vague. Either a component is a AE or a TE. I assume you mean AE, but that is unspecified in text.

Reply:
Probably should be 'target environment'

Reply:
Sure, that's great :-) but can we have the discussion about refining §3.4 in another PR, please?  This PR is just concerned with registering an OID for the CMW collection extension.

Reply:
Opened #46 to track this